eng/.docsettings.yml

```diff
@@ -25,7 +25,7 @@ language: net
 root_check_enabled: True
 
 required_readme_sections:
-  - "Azure .+ client library for .NET"
+  - ".+ client library for .NET"
```

Comment:
@weshaggard, I defer to you on whether this is the right approach to bypassing validations for the README for System.Net.ClientModel library.  I needed it to unblock the release pipeline for 1.0.0-beta.1 release, but happy to work with you on a better long-term solution.

Reply:
Awesome, I will do that instead.  Thank you!

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-net/pull/39335

Reply:
Arg, I messed this up in a merge conflict, but will fix before this PR merges.